backend/src/mocks/mock_api_interfaces.py

```diff
@@ -141,7 +171,29 @@ def mock_chat_response():
 
 def mock_user_info() -> dict:
     return {
-        "ID": "6b96666e-8b3a-4996-932d-3aa75c08c16f",
-        "name": "Michael Luch",
-        "agents": [mock_agent_interface(), mock_agent_alt()],
+        "userId": "6b96666e-8b3a-4996-932d-3aa75c08c16f",
+        "userName": "Michael Luch",
+        "userAgents": [mock_agent_interface(), mock_agent_alt()],
     }
+
+def mock_pinned_chats_list() -> dict:
+    return [
+			{
```

Comment:
Let's construct this from `mock_chat_interface()`. This way it will be clear that the structure in both places is identical.